### PR TITLE
fix(review-workflow): sanitize admin api requests

### PR DESCRIPTION
**Pull request by nathan-pichon**
### What does it do?

Sanitize inputs and outputs in review workflow controllers

### Why is it needed?

Security concern

### CODE REVIEW

**Comment by Boegie19** [comment]
@nathan-pichon You also want to sanitize the input aswell at least for findOne and it would not be bad to do it for all endpoints could save you with populating private fields

**Comment by jhoward1994** [comment]
I seem to get `TypeError: permissionChecker.sanitizeInput is not a function`

Do we need to use `sanitizeCreateInput` or `sanitizeUpdateInput` from the permission checker?

**Comment by nathan-pichon** [comment]
> I seem to get `TypeError: permissionChecker.sanitizeInput is not a function`
> 
> Do we need to use `sanitizeCreateInput` or `sanitizeUpdateInput` from the permission checker?

Yes sorry, I went back to the work I did on Friday and it wasn't finished. I thought I'd finished it, which is why I put it in review 😓

File: packages/core/admin/ee/server/controllers/workflows/index.js [hunk 1]
```diff
@@ -1,11 +1,32 @@
...
+ * @return { (Workflow) => SanitizedWorkflow }
+ */
+function sanitizeWorkflow({ strapi }, userAbility, type = 'output') {
+  const permissionChecker = strapi
+    .plugin('content-manager')
+    .service('permission-checker')
+    .create({ userAbility, model: WORKFLOW_MODEL_UID });
+
+  if (type === 'input') {
+    return (entity) => permissionChecker.sanitizeInput(entity);
```
Comment (Marc-Roig) [inline packages/core/admin/ee/server/controllers/workflows/index.js]: we have to use sanitizeInput for body params and sanitizeQuery for query params

**Review by Marc-Roig — APPROVED** [review]

**Pull request merged**